DAP.tex

```diff
@@ -483,39 +534,68 @@ \subsubsection{CALIB}
 \end{lstlisting}
 
 \subsubsection{TARGET}
-The TARGET parameter is a string-valued parameter that specifies the name of the target (e.g. the intention of the original science program or observation). The value is compared with the target\_name from the ObsCore data model.
+
+The TARGET parameter is a string-valued parameter that specifies the name of the target (e.g. the intention of the original science program or observation). The value is compared with the target\_name from the ObsCore data model. This parameter is case sensitive.
+
 
 \subsubsection{FORMAT}
-The FORMAT parameter specifies the  format returned by  the access  link.  The value is compared with the access\_format column from the ObsCore data model. This column describes the format of the response from the access\_url (see 3.1.3) so the values could be data file types (e.g. application/fits) or they could be the DataLink  MIME type (\cite{std:DataLink}, \cite{std:TSV}).  
 
-\subsection{RELEASEDATE}
+The FORMAT parameter specifies the  format returned by  the access  link.  The value is compared with the access\_format column from the ObsCore data model. This column describes the format of the response from the access\_url (see 3.1.3) so the values could be data file types (e.g. application/fits) or they could be the DataLink  MIME type (\cite{std:DataLink}, \cite{std:TSV}).  This parameter is case insensitive.
+
+\subsubsection{RELEASEDATE}
 The RELEASEDATE parameter specifies the range of release dates to be searched for data. 
 The limits are compared to the obs\_release\_date optional attribute of the ObsCore data model. 
 They are expressed as 2 ISO 8601 dates in the general case. A single value is searched for an exact match.
 As the obs\_release\_date attribute is optional, the service self description (\ref{sec:selfdesc})  informs the user of the availability of this parameter.
 RELEASEDATE queries for services not providing the release\_date attribute SHOULD provide an empty response.
 
+
+
+\subsubsection{RETRIEVEMODE (2 solutions)}
```

Comment:
It would have been better to keep the SIA->DAP transition changes separate from upgrades like this. The main issue I see is that implementation is going to be complex for anyone who relies on DAP -> DataLink to handle 1 product to N files cardinality... if that leads to "only some simple DAP services implement this" then that will be bad.

**Prefer:** extract this from PR and create an issue to discuss further.

Reply:
I dont' think it's the only change independant from SIA-> DAP transition. For example the introduction of RELEASEDATe and MOCs are others.

About the discussion on this point I hope to load today on the appropriate the Sydney discussion as I said last friday.

Reply:
I used an audio transcription on line facility and revisited it to assign  to participants their names. Its' there https://wiki.ivoa.net/internal/IVOA/InterOpMay2024DAL/DAL-2-notes or on the DAL sessions page of the last interop. Discussion was very rich. I see 4 points.

- initial controversy : Direct access to cutout using cutout (SODA) full URL in access_url or SODA service descriptors.  
- if cutout direct computation is chosen in combination with DataLink, is "this" item the cutout or the whole image ?
- can we reintroduce  with that change the different "INTERSECT" modes we had in SIAV1 instead of the unique OVERLAP mode.   Specially the COVERS mode where the full ROI is included in the image.
- When we force one output format for the datasets with the FORMAT= parameter how can this appear in combination with DataLink usage because access_format will contain "datalink" and not the format which has been chosen.

I personnaly think for the first controversy there is none : we can have the citout retrievmode forcing a soda  access_url  and the SODA service descriptor in the response for intercation with the service at the same time.

For the three other items I am going to write issues